project/src/common/find-movie-by-id.tsx

```diff
@@ -0,0 +1,14 @@
+import { Film } from '../types/film';
+import { Films } from '../mocks/films';
+
+export function FindMovieById(id: string | undefined): Film {
+  if (!id) {
+    throw new Error('Invalid id.');
```

Comment:
You better return undefined here, so your app will not stop if movie does not exist. Figure out a fallback for this case